utils/containerd/Makefile

```diff
@@ -1,15 +1,15 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=containerd
-PKG_VERSION:=1.4.12
+PKG_VERSION:=1.5.11
 PKG_RELEASE:=1
 PKG_LICENSE:=Apache-2.0
 PKG_LICENSE_FILES:=LICENSE
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://codeload.github.com/containerd/containerd/tar.gz/v${PKG_VERSION}?
-PKG_HASH:=85a531725f15e2d136131119d42af4507a5389e0947015152075c4c93816fb5c
-PKG_SOURCE_VERSION:=7b11cfaabd73bb80907dd23182b9347b4245eb5d
+PKG_HASH:=6a289406c1c0583763e5a9754e31a1eced55cd5f162a7bc2a3a315d5eb05c7a1
```

Comment:
I'll try and take a look - I can't figure out why it didn't fail locally?

Reply:
OK, I am absolutely baffled:
`make package/containerd/check FIXUP=1 V=s`
gives me 6a289 every time I replace PKG_HASH with skip and rerun it

How the hell is CI pulling a different package?  I can't blame the corporate Fortigate for this one, because I created this patchset when I was WFH and not behind the proxy.  I'm getting the same results on two completely different networks.

Reply:
After deleting the tarball and re-downloading, it's giving me your results now.  WTF?

Well, I'll commit it and hope this does not do something weird AGAIN.

Reply:
Committed with hash updated to 02b79

Reply:
Hmm, I got in this situation once also, we could probably add something to help avoid this. But I workaround it but cleaning.
IIRC, `make package/containerd/{clean,download,check} FIXUP=1 V=s` does it in one.